audioswitch/src/test/java/com/twilio/audioswitch/selection/AudioDeviceSelectorJavaTest.java

```diff
@@ -92,6 +93,16 @@ public void shouldAllowSelectingAudioDevice() {
         assertEquals(earpiece, javaAudioDeviceSelector.getSelectedAudioDevice());
     }
 
+    @Test
+    public void getVersion_shouldReturnValidSemverFormattedVersion() {
+        String semVerRegex =
+                "^([0-9]+)\\.([0-9]+)\\.([0-9]+)(?:-([0-9A-"
+                        + "Za-z-]+(?:\\.[0-9A-Za-z-]+)*))?(?:\\+[0-9A-Za-z-]+)?$";
+
+        assertNotNull(AudioDeviceSelector.VERSION);
```

Comment:
Can we stick to using one assertion framework to be consistent? Most of the tests use the `org.hamcrest.CoreMatchers` class.